tests/unit/test_repos_repo.py

```diff
@@ -874,6 +874,39 @@ def test_tree_required_sha(self):
 
         assert self.session.get.called is False
 
+    def test_update_label(self):
+        """Verify the request for updating a label."""
+        with mock.patch.object(Repository, 'label') as mocked_label:
```

Comment:
@sigmavirus24 ,
Here's a snippet from `update_label`: 
```python
label = self.label(name)                                                    
resp = False                                                                
if label:                                                                   
    upd = label.update                                                      
    resp = upd(new_name, color) if new_name else upd(name, color)           
    return resp  
```

Without mocking `Reposoitory.label`, the return value will be `False` (NullObject) and I won't be able to step into `if label:`.  Is there another way to get `self.label(name)` to be Truthy without mocking it  ?